docker/bin/cluster_up.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
```

Comment:
use `set -eu` at the start of all scripts to catch errors  (and then add proper handling of failing commands if they're okay to fail)